public/app/plugins/datasource/loki/querybuilder/parsing.ts

```diff
@@ -404,7 +404,7 @@ function handleRangeAggregation(expr: string, node: SyntaxNode, context: Context
 
   let match = getString(expr, node).match(/\[(.+)\]/);
   if (match?.[1]) {
-    params.push(match[1]);
+    params.unshift(match[1]);
```

Comment:
this is a bug fix that I noticed while working on this PR. Range is always the first parameter.